Sources_v3/StreamChatUI/ChatChannel/ChatMessage/ChatMessageContentView.swift

```diff
@@ -104,9 +111,9 @@ open class ChatMessageContentView<ExtraData: ExtraDataTypes>: View, UIConfigProv
             authorAvatarView.leadingAnchor.constraint(equalTo: leadingAnchor),
             authorAvatarView.bottomAnchor.constraint(equalTo: bottomAnchor),
             
-            messageReactionsView.topAnchor.constraint(equalTo: topAnchor),
+            reactionsBubble.topAnchor.constraint(equalTo: topAnchor),
             
-            messageBubbleView.trailingAnchor.constraint(equalTo: trailingAnchor).with(priority: .defaultHigh),
+            messageBubbleView.trailingAnchor.constraint(equalTo: trailingAnchor).with(priority: .required - 1),
```

Comment:
I found myself often in situation when I need `priority: 999`. Almost all our `with(Priority:)` needs to be 999
Maybe it's time to make:
```
extension UILayoutPriority {
    static var almostGuaranteed: UILayoutPriority { .init(rawValue: 999) }
}
```
Other possible names:
- `a999`: Will be used as `.with(priority: .a999)`. We all know what 999 means, so it's became domain knowledge. Leading `a` is required, because identifier cannot start with digit.
- `softRequired`
- your ideas?

Reply:
Great idea, let's do that!

I'd rather we preserve the `required` terminology: smth like `almostRequired`. It also would be nice to have a helper on `NSLayoutContraint`:

```swift
extension UILayoutPriority {
    static var almostRequired: UILayoutPriority { .init(rawValue: 999) }
}

extension NSLayoutContraint {
    var almostRequired: NSLayoutContraint {
      self.with(priority: .almostRequired)
    }
}
```